lib/src/io_client.dart

```diff
@@ -44,7 +45,7 @@ class IOClient extends BaseClient {
         headers[key] = values.join(',');
       });
 
-      return StreamedResponse(
+      return IOStreamedResponse(
```

Comment:
Let's change the return type on line 28 to `Future<IOStreamedResponse>`.

Reply:
👍 Updated in b699b87.